### DISCUSSION TITLE
Network VPN

### DISCUSSION THREAD

**Discussion by pmd1nh** [utmapp/UTM#3598] [UTM Help (macOS)]
Hi,

How can I connect to my work network (via VPN) from the guest Ubuntu 20.0.4 (UTM version 3.0.4 on Intel macOS Big Sur)?  the network works fine before vpn.  After connected to remote network (via Cisco VPN) from the macOS host, nothing is working on the guest.  I even try to manually add the static IP address of the DNS remote nameserver but doesn't help.

Thanks!

**Comment by fduser1** [comment]
Network mode `Emulated VLAN` worked for me using UTM 4.1.5 on M1 macOS Ventura.

If you want to SSH or connect from the host to guest, then you can configure a *Port Forward* rule using port numbers only.

If you want to connect the other way around, then you can use host IP address.
